.docker/plugins/cinder/config.json

```diff
@@ -0,0 +1,136 @@
+{
+      "Args": {
+        "Description": "",
+        "Name": "",
+        "Settable": null,
+        "Value": null
+      },
+      "Description": "REX-Ray for Cinder Persistent Disks",
+      "Documentation": "https://github.com/codedellemc/rexray/.docker/plugin/cinder",
```

Comment:
Hi @sebglon,

Please correct the URI so that:
* `plugins` is plural
* The URI includes `tree/master`

```
https://github.com/codedellemc/rexray/.docker/plugin/cinder
```

should be

```
https://github.com/codedellemc/rexray/tree/master/.docker/plugins/cinder
```